src/main/java/com/exadel/sandbox/controller/EmployeeController.java

```diff
@@ -0,0 +1,53 @@
+package com.exadel.sandbox.controller;
+
+import com.exadel.sandbox.dto.EmployeeDto;
+import com.exadel.sandbox.entities.Employee;
+import com.exadel.sandbox.mapper.EmployeeMapper;
+import com.exadel.sandbox.service.EmployeeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("employee")
+@RequiredArgsConstructor
+public class EmployeeController {
+
+    private final EmployeeService employeeService;
+    private final EmployeeMapper mapper;
+
+    @GetMapping("list")
+    List<EmployeeDto> getEmployees() {
+        List<Employee> employees = employeeService.getEmployees();
+        return employees.stream().map(mapper::toDto).collect(Collectors.toList());
```

Comment:
The controller needed only to accept the request and transfer it to the server, (possibly also for logs and for security).
Please move this logic to the service and leave just employeeService.getService() and check other places in controller